src/server/queue_func.c

```diff
@@ -121,7 +121,7 @@ que_alloc(char *name)
 	CLEAR_HEAD(pq->qu_jobs);
 	CLEAR_LINK(pq->qu_link);
 
-	snprintf(pq->qu_qs.qu_name, PBS_MAXQUEUENAME, "%s", name);
+	snprintf(pq->qu_qs.qu_name, PBS_MAXQUEUENAME + 1, "%s", name);
```

Comment:
Instead of using PBS_MAXQUEUENAME + 1 in snprintf, please use sizeof(pq->qu_qs.qu_name). This way even if the value of macro changes we no need to change this piece of code.